crates/eip7547/src/summary.rs

```diff
@@ -82,7 +82,7 @@ pub struct InclusionListSummaryEntryV1 {
     /// The address of the inclusion list entry.
     pub address: Address,
     /// The nonce of the inclusion list entry.
-    pub nonce: u64,
+    pub nonce: U64,
```

Comment:
can we use u64 and serde_with alloy_serde::u64_hex for all U64?

Reply:
whoops, yeah